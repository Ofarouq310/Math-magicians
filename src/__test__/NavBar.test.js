import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../components/NavBar';

describe('check Navbar', () => {
  const tree = renderer.create(<Router><NavBar /></Router>).toJSON();
  test('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
