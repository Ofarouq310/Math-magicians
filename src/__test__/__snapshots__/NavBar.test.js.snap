// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check Navbar renders correctly 1`] = `
<nav>
  <h1>
    Math Magicians
    <span>
       
    </span>
  </h1>
  <ul
    className="navLink"
  >
    <li
      className="navItem"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <span>
          Home
        </span>
      </a>
    </li>
    <span>
      |
    </span>
    <li
      className="navItem"
    >
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <span>
      |
    </span>
    <li
      className="navItem"
    >
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
